packages/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,232 @@
+part of firebase_remote_config;
+
+/// The entry point for accessing Remote Config.
+///
+/// You can get an instance by calling [RemoteConfig.instance]. Note
+/// [RemoteConfig.instance] is async.
+class RemoteConfig extends ChangeNotifier {
+  static const MethodChannel _channel =
+      const MethodChannel('plugins.flutter.io/firebase_remote_config');
+
+  static const String defaultValueForString = '';
+  static const int defaultValueForInt = 0;
+  static const double defaultValueForDouble = 0.0;
+  static const bool defaultValueForBool = false;
+
+  static const String _fetchFailedThrottledKey = 'FETCH_FAILED_THROTTLED';
+  static const String _lastFetchTimeKey = 'LAST_FETCH_TIME';
```

Comment:
I think we should either use constants for all the keys (e.g. `'IN_DEBUG_MODE'`) or none of them. Also I think the keys should be lower case to match other first-party plugins.

Reply:
Done, removed constants for keys